.github/workflows/gradle-build.yml

```
@@ -24,4 +24,24 @@
+      run: |
+        echo "VERSION_CODE=$(grep -o "versionCode\s\+\d\+" app/build.gradle | awk '{ print $2 }')" >> $GITHUB_ENV
+        echo "APP_ID=$(awk '/^\s*applicationId/ { gsub(/"/, "", $2); print $2 }' app/build.gradle)" >> $GITHUB_ENV
+        echo "Version code is: " + ${{ env.VERSION_CODE }}
+
+    - name: Check version code
+      if: always()
+      env:
+        VERSION_CODE: ${{ secrets.VERSION_CODE }}
+      run: |
+        echo "Check version code 2: $VERSION_CODE"
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)